### PR TITLE
Make sure realm is available from session when migrating to 23

### LINKED ISSUE #25183: NullPointerException thrown for UPConfig.getGroups()
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

```sh
ERROR [org.keycloak.services.error.KeycloakErrorHandler] (executor-thread-7) Uncaught server error: java.lang.NullPointerException: Cannot invoke "org.keycloak.representations.userprofile.config.UPConfig.getGroups()" because "config" is null
	at org.keycloak.userprofile.config.UPConfigUtils.validateAttributes(UPConfigUtils.java:112)
	at org.keycloak.userprofile.config.UPConfigUtils.validate(UPConfigUtils.java:95)
	at org.keycloak.userprofile.DeclarativeUserProfileProvider.lambda$createUserDefinedProfileDecorator$8(DeclarativeUserProfileProvider.java:523)
	at java.base/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708)
	at org.keycloak.userprofile.DeclarativeUserProfileProvider.configureUserProfile(DeclarativeUserProfileProvider.java:184)
	at org.keycloak.userprofile.AbstractUserProfileProvider.createUserProfile(AbstractUserProfileProvider.java:328)
	at org.keycloak.userprofile.AbstractUserProfileProvider.create(AbstractUserProfileProvider.java:205)
	at org.keycloak.services.resources.admin.UserResource.getUser(UserResource.java:325)
	at org.keycloak.services.resources.admin.UserResource$quarkusrestinvoker$getUser_7e789726117d7cfd3ef1d9ddd01596d8e6add5b0.invoke(Unknown Source)
	at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:145)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:576)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)
```

### Version

23.0.1

### Expected behavior

No errors when attempting to view User Detail pages

### Actual behavior

When trying to go into the User details for some Realms, getting a "Network response was not OK". The suggesting reload of the page does not resolve.

image

### How to Reproduce?

Try to navigate into a user detail when the error is present. 

### Anything else?

Doesn't appear to affect all realms so possibly related to some partial migration step?

Installation was updated automatically using the Keycloak Operator.

### PR DESCRIPTION
**Pull request by pedroigor**
Closes #25183

### CODE REVIEW

File: testsuite/integration-arquillian/tests/pom.xml [hunk 1]
```diff
@@ -1534,6 +1534,8 @@
                                     <auth.server.migration>true</auth.server.migration>
                                     <keycloak.migration.home>${containers.home}/auth-server-migration</keycloak.migration.home>
                                     <migration.import.props.previous>${migration.import.props.previous}</migration.import.props.previous>
+                                    <auth.server.feature>${auth.server.feature}</auth.server.feature>
```
Comment (shawkins) [inline testsuite/integration-arquillian/tests/pom.xml:1537]: Do you mean for this key to refer to itself, and be the same as the key on the next line?

**Review by shawkins — APPROVED** [review]
LGTM, just one question about the maven property.

**Review by stianst — APPROVED** [review]
Approving based on review in https://github.com/keycloak/keycloak/pull/25313

File: testsuite/integration-arquillian/tests/pom.xml (hunk 1 above)
Comment (pedroigor) [reply testsuite/integration-arquillian/tests/pom.xml:1537]: Actually, only the next line is needed. This one was an attempt to make it configurable but for migration we just want to enable it by default.

The value from the last one is what we need and the value actually set.

Will send a PR fixing this ... Tks.

**Pull request merged**